engine/test-src/org/pentaho/di/trans/steps/groupby/GroupByTest.java

```diff
@@ -98,4 +104,27 @@ public void testProcessRow() throws KettleException {
 
     assertTrue( groupBySpy.getOutputRowSets().get( 0 ).isDone() );
   }
+
+  @Test
+  public void testGetFields() {
```

Comment:
Please also test the TYPE_GROUP_STANDARD_DEVIATION and TYPE_GROUP_MEDIAN, from the GroupByMeta getFields() method.